### PR TITLE
[FORSETI RELEASE] Update Forseti version to v2.24.1

### PR DESCRIPTION
**Pull request by red2k18**
[FORSETI RELEASE] Update Forseti version to v2.24.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -316,7 +322,10 @@ Version 4.0.0 is a backwards-incompatible release. Please see the [upgrade instr
 [v4.3.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v4.2.1...v4.3.0
 [v5.0.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v4.3.0...v5.0.0
 [v5.1.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v5.0.0...v5.1.0
+[v5.1.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v5.0.0...v5.1.1
```
Comment (gkowalski-google) [inline CHANGELOG.md]: Change to 5.1.1

File: CHANGELOG.md [hunk 2]
```diff
@@ -316,7 +322,10 @@ Version 4.0.0 is a backwards-incompatible release. Please see the [upgrade instr
...
+[v5.1.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v5.1.1...v5.1.2
```
Comment (gkowalski-google) [inline CHANGELOG.md]: Change to 5.1.2

File: CHANGELOG.md [hunk 3]
```diff
@@ -316,7 +322,10 @@ Version 4.0.0 is a backwards-incompatible release. Please see the [upgrade instr
...
 
+[#476]: https://github.com/forseti-security/terraform-google-forseti/pull/477
```
Comment (gkowalski-google) [inline CHANGELOG.md]: Fix index, should be `[#477]` and make sure link above is updated

Comment (red2k18) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (red2k18) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (red2k18) [reply CHANGELOG.md]: Done

**Pull request closed without merging**